GetIntoTeachingApi/Models/Location.cs

```diff
@@ -9,8 +9,14 @@ namespace GetIntoTeachingApi.Models
 {
     public class Location
     {
+        public static readonly Regex OutwardOrFullPostcodeRegex = new Regex(
+            $"^({OutwardPostcodePattern}|{FullPostcodePattern})$", RegexOptions.IgnoreCase);
         public static readonly Regex PostcodeRegex = new Regex(
-            @"^([A-Z][A-HJ-Y]?\d[A-Z\d]? ?\d[A-Z]{2}|GIR ?0A{2})$", RegexOptions.IgnoreCase);
+            $"^({FullPostcodePattern})$", RegexOptions.IgnoreCase);
+

```

Comment:
Not sure if I have made this too convoluted, with the carets, dollars, parenthesis, and or pipe in the Regex object, but the rest in the const strings

Reply:
I think it's fine - I would maybe opt for interpolation in your `FullPostcodePattern`, but just my preference:

```
$"{OutwardPostcodePatter} ?{InwardPostcodePattern}"
```

Reply:
Ah, yeah it won't allow that because they are const. I can make them static readonly, I think that might work